hw/mcu/dialog/da14699/include/mcu/mcu.h

```diff
@@ -150,6 +150,8 @@ void mcu_gpio_exit_sleep(void);
 #define MCU_MEM_QSPIF_M_END_ADDRESS     (0x18000000)
 #define MCU_MEM_SYSRAM_START_ADDRESS    (0x20000000)
 #define MCU_MEM_SYSRAM_END_ADDRESS      (0x20080000)
+#define MCU_MEM_QSPIF_M_RANGE_ADDRESS(x) \
+    ((uint32_t)x >= (uint32_t)MCU_MEM_QSPIF_M_START_ADDRESS && (uint32_t)x <= (uint32_t)MCU_MEM_QSPIF_M_END_ADDRESS)
```

Comment:
Might be better to add parenthesis around `x`

Reply:
done